components/input.css

```diff
@@ -0,0 +1,84 @@
+.input-field{
+    width: 70%;
+    padding: 0.5rem 1rem;
+    border-radius: 5px;
+    outline: none;
+    border: 1px solid black;
+}
+
+.disabled-input{
+    pointer-events:none;
+    background-color: rgb(231, 231, 231);
+}
+
+.input-err{
+    width: 70%;
+}
```

Comment:
You can name this error instead of err. You need to give more descriptive names.